docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -0,0 +1,84 @@
+# Common look and feel of apps
+
+## Introduction
+
+Good user experience is significant for a successful solution of apps. An author of a single app aims to balance a good default look that matches other apps and to have distinctive features that are both easy to use and pleasing to the eye of the end user.
+
+Starcounter offers a layered approach that allows app authors to adapt to a common look and feel for the typical parts of the UI and to provide their own components for the unique parts. Solution owners can theme or replace the look and feel without changing of the app source code.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a standards-based layered approach that separates the responsibilities of the app author, the design system, and the solution owner. 
+
+The full design consists of:
+
+1. Content defined by native HTML
+2. Common UI patterns and components of Uniform design system
+3. Theme configuration through custom CSS properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Content defined by native HTML
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the native semantic HTML elements that can be rendered on the screen directly or with their appearance changed in the presentation layer.
+
+### Common UI patterns and components of Uniform design system
+
+As the basic way for providing a common look and feel in the presentation layer, Starcounter provides Uniform design system. It consists of custom CSS properties, CSS reset and a library of patterns, and components.
```

Comment:
>  It consists of custom CSS properties, CSS reset and a library of patterns, and components.

The comma placement is weird here. I usually use an oxford comma, with that, it should look like this:

>   It consists of custom CSS properties, CSS reset, and a library of patterns and components.

You could also skip the last comma:

>  It consists of custom CSS properties, CSS reset and a library of patterns and components.

Also, what does it mean to have a library of patterns? When I think patterns, I think OO patterns which are abstract descriptions of how to structure your code. What do we mean with patterns here? 

Is it a conscious choice to not use a definitive article for "Uniform design system"? Similar cases use definitive articles:
- [Carbon design system](http://www.carbondesignsystem.com/getting-started/designers): "The Carbon Design Kit is a living, breathing document that contains..."
- [Lighting design system](https://www.lightningdesignsystem.com/): "The Lightning Design System enables you to build..."

Reply:
> The comma placement is weird here. I usually use an oxford comma, with that, it should look like this

Thanks! I got tricked by Grammarly.

> When I think patterns, I think OO patterns which are abstract descriptions of how to structure your code. What do we mean with patterns here?

User Interface Design patterns are recurring solutions that solve common design problems. Design patterns are standard reference points for the experienced user interface designer. [User interface design patters](http://ui-patterns.com/patterns).

I will clarify in the text.

> Is it a conscious choice to not use a definitive article for "Uniform design system"? Similar cases use definitive articles:

Good catch! Yes, that *was* conscious, I wanted it to appear more flexible by that. but I agree this is clumsy. I will change to definitive.